airflow/models.py

```diff
@@ -708,8 +718,10 @@ def generate_command(dag_id,
                          task_id,
                          execution_date,
                          mark_success=False,
-                         ignore_dependencies=False,
+                         ignore_all_deps=False,
```

Comment:
It will be nice when we start pass the dependency context object around and get simpler function signatures